contrib/helm/harbor/templates/registry/registry-cm.yaml

```diff
@@ -8,7 +8,7 @@ data:
   config.yml: |+
     version: 0.1
     log:
-      level: info
+      level: {{ .Values.registry.logLevel }}
```

Comment:
There is no default value for .Values.registry.logLevel in values.yaml. Need to add it.

Reply:
added in db67636fdd7debb1a88cf7b707f553165ef1d702